h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -318,6 +318,10 @@ public enum MissingValuesHandling {
     public double _init_dispersion_parameter = 1.0;
     public boolean _fix_dispersion_parameter = false;
     public boolean _build_null_model = false;
+    public double _tweedie_epsilon = 8e-27; // paper suggests 8e-17
```

Comment:
I'm not sure if there is a typo or not.
The paper suggests `exp(−37) ≈ 8 × 10^−17` but we use 8e-27 instead.

Reply:
changed back to 8e-17